bigquery/query_join_your_data.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: default
+title: Joining with Your Own Data
+nav_order: 4
+parent: DC to BQ Sample Queries
+grand_parent: BigQuery
+---
+
+# Query Category: Joining with Your Own Data
+
+* Using FIPS code:
```

Comment:
Add a link to FIPS? Perhaps this: https://www.census.gov/library/reference/code-lists/ansi.html

Reply:
Done